scripts/kind-e2e/e2e.sh

```diff
@@ -122,6 +122,38 @@ function setup_broker() {
     SUBMARINER_BROKER_TOKEN=$(kubectl --context=cluster1 -n ${SUBMARINER_BROKER_NS} get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='${SUBMARINER_BROKER_NS}-client')].data.token}"|base64 --decode)
 }
 
+function install_subm_in_cluster1() {
+    if kubectl --context=cluster1 wait --for=condition=Ready pods -l app=submariner-engine -n submariner --timeout=60s > /dev/null 2>&1; then
+            echo Submariner already installed, skipping submariner helm installation...
+            update_subm_pods cluster1
+        else
+            echo Installing submariner on cluster1...
+            worker_ip=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster1-worker | head -n 1)
+            #kubectl --context=cluster1 label node cluster1-worker "submariner.io/gateway=true" --overwrite
```

Comment:
Can you remove this line?

Reply:
Commented line highlights that the label is not set. I will write a comment instead before the function.

Reply:
Done